settings.gradle

```diff
@@ -31,7 +31,6 @@ micronautBuild {
     importMicronautCatalog("micronaut-multitenancy")
     importMicronautCatalog("micronaut-testresources")
     importMicronautCatalog("micronaut-validation")
-    importMicronautCatalog("micronaut-logging")
```

Comment:
Was seeing this info when running tests: 
> Version catalog 'mnLogging' can be automatically imported. You can remove it from settings.gradle(.kts) file

So removed it from settings.gradle